src/firmware_main_loop.c

```diff
@@ -30,28 +35,59 @@ void firmware_main_loop(void)
     // This starts the async orientation screen workflow, which is processed by the loop below.
     orientation_screen();
 
+    const uint8_t* hww_data = NULL;
+    const uint8_t* u2f_data = NULL;
```

Comment:
guard u2f things with `#if APP_U2F == 1`, it's confusing if it's guarded below but not here.